charts/seed-monitoring/charts/prometheus/templates/config.yaml

```
@@ -570,3 +570,6 @@ data:
       metric_relabel_configs:
 {{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.blackboxExporter | indent 8 }}
 
+    - job_name: 'vpa'
```

Use Kubernetes [service discovery](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kubernetes_sd_config) instead of static configs. 
```suggestion
    - job_name: 'vpa'
      kubernetes_sd_configs:
      - role: endpoints
        namespaces:
          names: [{{ .Release.Namespace }}]
      relabel_configs:
      - source_labels:
        - __meta_kubernetes_service_name
        - __meta_kubernetes_endpoint_port_name
        action: keep
        regex: vpa-exporter;metrics
```